docs/docs/en/confluent/security.md

```diff
@@ -85,7 +85,7 @@ from faststream.security import SASLPlaintext
 
 security = SASLPlaintext(
     username="admin",
-    password="password",
+    password="password",  # pragma: allowlist secret
```

Comment:
Can we do not check `.md` files by bandit or make this file includable with cutting off the comment (like others in this section)? I don't like an idea to show such comments in our documentation